source/PasswordManager.cpp

```diff
@@ -0,0 +1,15 @@
+#include <iostream>
+
+#include "../headers/PasswordManager.h"
```

Comment:
Since we have "include_directories(headers)" in the main CMakeLists, simple "PasswordManager.h" will work.